core/services/relay/dydx/config_digester.go

```diff
@@ -0,0 +1,48 @@
+package dydx
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+const ConfigDigestPrefixDydx types.ConfigDigestPrefix = 4
```

Comment:
Can we add a TODO to move this into libocr?

Reply:
I'll send out another PR in libocr repo.

Reply:
Let's leave as a TODO for now, it will take some time for the new libocr to get released

Reply:
ACK